zstacklib/zstacklib/gpu/base.py

```
@@ -0,0 +1,480 @@
+        Normalize PCI address to standard format.
+        
+        Converts "00000000:3B:00.0" to "0000:3B:00.0"
+        """
+        pci_address = pci_address.strip().lower()
+        # Remove 8-char domain prefix (e.g., 00000000:3B:00.0 -> 0000:3B:00.0)
+        if len(pci_address.split(':')[0]) == 8:
+            pci_address = pci_address[4:]
+        return pci_address
+    
+    @staticmethod
+    def parse_unit_value(value, target_unit=None):
```

Ruff (0.14.13)

Unused static method argument: `target_unit`

(ARG004)